src/FSharpVSPowerTools.Logic/ProjectSystem/VSLanguageService.fs

```diff
@@ -338,4 +348,17 @@ type VSLanguageService
     /// This value is used for testing when VS lex cache isn't available
     member internal __.SkipLexCache 
         with get () = skipLexCache
-        and set v = skipLexCache <- v
\ No newline at end of file
+        and set v = skipLexCache <- v
+
+
+    member __.MakePointInDocument (point: SnapshotPoint option) filename =
```

Comment:
I'm thinking as extension member to SnapshotPoint (which is object we will use in most cases to construct PointInDocument)?